src/snapred/backend/data/LocalDataService.py

```diff
@@ -363,6 +374,27 @@ def _getVersionFromCalibrationIndex(self, runId: str):
             version = latestCalibration.version
         return version
 
+    def _getVersionFromNormalizationIndex(self, runId: str):
```

Comment:
Are Normalizations not coupled with Diffraction Calibrations?  
I had thought we were just going to save them under the relevant calibration folder.

Reply:
checking in the tests, they are saved in the same folder, next to the calibration records

Reply:
Yes, they are supposed to be saved in the same calibration version directory.

Reply:
This is where it tries to save to from the GUI.  These folders contain calibration parameters.  We're up to v50 for this state's directory.  It errors-out for me, but would for for a CIS  
![image](https://github.com/neutrons/SNAPRed/assets/52183986/19f58a8b-a488-4bf6-a21c-40a0459e6914)